src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -44,30 +44,11 @@ public void testDoGet() throws IOException, ServletException {
     Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
   }
 
-  public void testDoPost_ExistingUser() throws IOException, ServletException {
-
-    User user =
-        new User(
-            UUID.randomUUID(),
-            "test username",
-            "$2a$10$.e.4EEfngEXmxAO085XnYOmDntkqod0C384jOR9oagwxMnPNHaGLa",
-            Instant.now(), "test_aboutMe");
-
-    Mockito.when(mockRequest.getParameter("username")).thenReturn("test username");
-    Mockito.when(mockRequest.getParameter("password")).thenReturn("test password");
-
-    UserStore mockUserStore = Mockito.mock(UserStore.class);
-    Mockito.when(mockUserStore.isUserRegistered("test username")).thenReturn(true);
-    Mockito.when(mockUserStore.getUser("test username")).thenReturn(user);
-    profileServlet.setUserStore(mockUserStore);
-
-    HttpSession mockSession = Mockito.mock(HttpSession.class);
-    Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
+  @Test
+  public void testDoPost_UserNotLoggedIn() throws IOException, ServletException {
+    Mockito.when(mockSession.getAttribute("user")).thenReturn(null);
 
     profileServlet.doPost(mockRequest, mockResponse);
-
```

Comment:
What is this test verifying?

Reply:
This PR was solely to get a MVP working and to formulate a prototype test that compiles and run, I was going to modify the tests in the next PR.

Reply:
On second thought I guess I could just finished the tests in this PR?